zio-schema-derivation/shared/src/main/scala-3/zio/schema/Derive.scala

```diff
@@ -14,7 +14,9 @@ object Derive {
     DeriveInstance().deriveInstance[F, A](deriver, schema, top = true)
 }
 
-private case class DeriveInstance()(using val ctx: Quotes) extends ReflectionUtils(ctx) {
```

Comment:
Need to move away from extending `ReflectionUtils`, or it will get `The type of a class parent cannot refer to constructor parameters, but zio.schema.ReflectionUtils[(DeriveInstance.this.ctx : quoted.Quotes)] refers to ctx`